File: app/src/main/java/com/startapp/quicknoteeasy/noteClass.java
package com.startapp.quicknoteeasy;

public class noteClass {
    String noteText;
    String noteTitle;
    String noteCreationDate;
    String imageNote; //refer to address in gallery

    int Id;


    public noteClass(String noteText, String noteTitle, String noteCreationDate, String imageNote) {
        this.noteText = noteText;
        this.noteTitle = noteTitle;
        this.noteCreationDate = noteCreationDate;
        this.imageNote = imageNote;
    }

    public noteClass(String noteTitle, String noteCreationDate) {
        this.noteTitle = noteTitle;
        this.noteCreationDate = noteCreationDate;
    }

    public String getImageNote() {
        return imageNote;
    }

    public void setImageNote(String imageNote) {
        this.imageNote = imageNote;
    }

    public String getNoteText() {
        return noteText;
    }

    public void setNoteText(String noteText) {
        this.noteText = noteText;
    }

    public String getNoteTitle() {
        return noteTitle;
    }

    public void setNoteTitle(String noteTitle) {
        this.noteTitle = noteTitle;
    }

    public String getNoteCreationDate() {
        return noteCreationDate;
    }

    public void setNoteCreationDate(String noteCreationDate) {
        this.noteCreationDate = noteCreationDate;
    }

}

File: app/src/main/java/com/example/weather/Classes/NoteClass.kt
package com.example.weather.Classes

class noteClass {
    var noteText: String? = null
    var noteTitle: String
    var noteCreationDate: String
    var imageNote //refer to address in gallery
            : String? = null
    var Id = 0

    constructor(
        noteText: String?,
        noteTitle: String,
        noteCreationDate: String,
        imageNote: String?
    ) {
        this.noteText = noteText
        this.noteTitle = noteTitle
        this.noteCreationDate = noteCreationDate
        this.imageNote = imageNote
    }

    constructor(noteTitle: String, noteCreationDate: String) {
        this.noteTitle = noteTitle
        this.noteCreationDate = noteCreationDate
    }
}